spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableProxyFactory.java

```diff
@@ -335,29 +185,13 @@ public void unbindOutputs(ChannelBindingService channelBindingService) {
 		}
 	}
 
-	/**
-	 * Holds information about the channels exposed by the interface proxy, as well as
-	 * their status.
-	 *
-	 */
-	static class ChannelHolder {
-
-		private MessageChannel messageChannel;
-
-		private boolean bindable;
-
-		public ChannelHolder(MessageChannel messageChannel, boolean bindable) {
-			this.messageChannel = messageChannel;
-			this.bindable = bindable;
-		}
-
-		public MessageChannel getMessageChannel() {
-			return this.messageChannel;
-		}
-
-		public boolean isBindable() {
-			return this.bindable;
-		}
+	@Override
+	public Set<String> getInputs() {
```

Comment:
should this method be called `getInputNames()` instead? (same question for `getOutputs`)